lib/baza-rb.rb

```diff
@@ -696,41 +696,50 @@ def checked(ret, allowed = [200])
     allowed = [allowed] unless allowed.is_a?(Array)
     mtd = (ret.request.original_options[:method] || '???').upcase
     url = ret.effective_url
+
     if ret.return_code == :operation_timedout
       msg = "#{mtd} #{url} timed out in #{ret.total_time}s"
       @loog.error(msg)
       raise TimedOut, msg
     end
+
     log = "#{mtd} #{url} -> #{ret.code} (#{format('%0.2f', ret.total_time)}s)"
+
     if allowed.include?(ret.code)
       @loog.debug(log)
       return ret
     end
-    @loog.debug("#{log}\n  #{(ret.headers || {}).map { |k, v| "#{k}: #{v}" }.join("\n  ")}")
+
+    # Log **all response headers**
```

Comment:
@paveltrty we don't use empty lines inside method bodies and we also don't use comments